src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/DiskRPTests.ps1

```diff
@@ -69,17 +69,19 @@ function Test-Disk
         Assert-AreEqual $false $disk.EncryptionSettings.Enabled;
 
         # Grant access test
-        Grant-AzureRmDiskAccess -ResourceGroupName $rgname -DiskName $diskname -Access $access -DurationInSecond 5;
-        Revoke-AzureRmDiskAccess -ResourceGroupName $rgname -DiskName $diskname;
+        $job = Grant-AzureRmDiskAccess -ResourceGroupName $rgname -DiskName $diskname -Access $access -DurationInSecond 5 -AsJob;
+        $job | Wait-Job;
+        $job = Revoke-AzureRmDiskAccess -ResourceGroupName $rgname -DiskName $diskname -AsJob;
+        $job | Wait-Job;
```

Comment:
Can you add asserts for Grant-AzureRmDiskAccess and Revoke-AzureRmDiskAccess to ensure they have succeeded?